src/components/Audioplayer/RepeatDropdown/index.js

```diff
@@ -107,7 +119,10 @@ export default class RepeatButton extends Component {
 
     return (
       <Col md={12} style={{paddingTop: 15}}>
-        Ayah: <br />
+        <LocaleFormattedMessage
+          id={ "player.currentAyah" }
```

Comment:
No, Ill love to learn and follow best practices. So we don't need {} for string literals